pennylane/_device.py

```diff
@@ -413,7 +438,9 @@ def supports_operation(self, operation):
             if operation.endswith(Operation.string_for_inverse):
```

Comment:
I will be glad to get rid of this once Device no longer sends operation strings!